app/lib/features/chat/widgets/custom_input.dart

```diff
@@ -355,23 +355,12 @@ class _TextInputWidgetConsumerState extends ConsumerState<_TextInputWidget> {
             return ListTile(
               leading: Consumer(
                 builder: (context, ref, child) {
-                  final mentionProfile =
-                      ref.watch(memberProfileProvider(roomMember['link']));
-                  return mentionProfile.when(
-                    data: (profile) {
-                      return ActerAvatar(
-                        mode: DisplayMode.User,
-                        uniqueId: roomMember['link'],
-                        avatar: profile.getAvatarImage(),
-                        displayName: title,
-                        size: profile.hasAvatar() ? 18 : 36,
-                      );
-                    },
-                    error: (e, st) => Text(
-                      'Error loading avatar due to ${e.toString()}',
-                      textScaleFactor: 0.2,
-                    ),
-                    loading: () => const CircularProgressIndicator(),
+                  return mentionPorfileBuilder(
```

Comment:
Minor typo here.

Reply:
And also wrap this with ```Consumer```. You could see how you did it with ```replyBuilder``` :)

Reply:
Fixed by https://github.com/acterglobal/a3/pull/855/commits/4239d5e28ce9904e7638301fb7f94b861d1481ae